index.js

```diff
@@ -9,65 +9,100 @@ const newMember = require("./data/new-member");
  * @param {Object} obj -
  * @return {number} The number of keys in the object
  */
-function numberOfKeys(obj) {}
+function numberOfKeys(obj) {
+  const keys = _.keysIn(yogaClasses[0]);
+  return keys.length;
+}
 
 /**
  * Remove the falsy values in a numbers array and return the sum
  * @param {numbers[]} array - An array of numbers that can also contain some falsy values
  * @return {number} The sum of the numbers in an array
  */
-function sumNumbers(array) {}
+function sumNumbers(array) {
+  const noFalsy = _.compact(simpleArray);
+  const sumTruthy = _.sum(noFalsy);
+  return (sumTruthy);
+}
 
 /**
  * Convert a two-dimensional array of new member data (each inner array having two values: the first being the key, the second being the value) into an object
  * @param {Array[]} member -
  * @return {number} The sum of the numbers in an array
  */
-function newMemberArrayToObject(member) {}
+function newMemberArrayToObject(member) {
+  const newMemberObject= _.fromPairs(newMember)
+  return(newMemberObject);
+}
 
 /**
  * Return an array of objects that grouped by instructors from the classes array of objects
  * @param {Object[]} collection - an array of yoga class objects
  * @return {Object[]} - the reshaped collection where the classes are grouped by instructor name
  */
-function groupClassByInstructor(collection) {}
+function groupClassByInstructor(collection) {
```

Comment:
The 'collection' parameter should be used instead of the global 'yogaClasses'. This will make the function versatile and usable with different data sets.